File: SM/Algoritmos, Estrutura de Dados e Linguagens/Exercises/0000000027.pas
PROGRAM ex0000000027;
USES CRT;
VAR x,y,media : Real;
BEGIN
    CLRSCR;
    readln(x);readln(y);
    media := (x+y) / 2;
    IF (media >= 0.0) AND (media < 3.0) THEN 
        writeln('Reprovado')
    ELSE IF (media >= 3.0) AND (media < 7.0) THEN 
        writeln('Exame')
    ELSE
        writeln('Aprovado');
END.

File: SM/Algoritmos, Estrutura de Dados e Linguagens/Exercises/0000000027.c
#include <stdio.h>

int main(){
    float x,y,z;
    scanf("%f%f",&x,&y);
    z = (x+y) / 2;
    if(z >= 0 && z < 3)
        printf("Reprovado.");
    else if (z >= 3 && z < 7)
        printf("Exame.");
    else
        printf("Aprovado.");
    
}